legate/driver/command.py

```diff
@@ -127,7 +127,15 @@ def cmd_nsys(
 
     opts: CommandPart = ("nsys", "profile", "-t", targets, "-o", log_path)
     opts += tuple(extra)
-    if "-s" not in extra:
+
+    has_sample_flag = False
+    for option in extra:
+        flag = option.split("=")[0]
+        if flag == "-s" or flag == "--sample":
+            has_sample_flag = True
+            break
```

Comment:
This seems ok. This might also be OK

```python
has_sample_flag = any(x.startswith(("-s", "--sample")) for x in extra)
```
The "right" way to do this would be to define a separate `argparse` parser and call it manually with `extra` but that's probably excessive for this simple situation.

Reply:
So I thought about that (not exactly that, but similar). The concern there was that there would be flags like "--sample-period" that would return true, but wouldn't actually be the `--sample` flag.  I guess `-s` is safe to check that way.

Reply:
Definitely a valid concern, I would leave it as is then

Reply:
This does seem amenable to a basic unit year in `test_command.py`